docs/environment.yml

```diff
@@ -1,118 +1,208 @@
-name: cityenergyanalyst_docs
```

Comment:
I don't understand why you're changing this. Have you tested it to work on the readthedocs? I'd prefer this to be a separate issue (updating the environment.yml for read-the-docs) - the RTD build system is not a windows system and therefore the environment won't have the exact same versions (your changes add vc9_1 versions of some modules which don't exist in ubuntu as the compiler doesn't exist).

Reply:
oh got you. I need your help with making this run